### DISCUSSION TITLE
Packages/com.unity.burst/manual/docs/DebuggingAndProfiling

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#530] [Notes]
# Packages/com.unity.burst/manual/docs/DebuggingAndProfiling

https://docs.unity3d.com/Packages/com.unity.burst@latest/index.html?subfolder=%2Fmanual%2Fdocs%2FDebuggingAndProfiling.html

**Comment by radiatoryang** [comment]
To debug a Job, really the easiest thing to do is to just comment out `[BurstCompile]` from the Job temporarily. The Job will run in C# mode instead of Burst. (see https://forum.unity.com/threads/console-logging-debug-log-equivalent-for-dots-inside-jobs-execute.807843/)... Of course this won't work if you're debugging Burst specifically, but it's good for general bug-fixing.

There's special support for `Debug.Log()` -- see https://docs.unity3d.com/Packages/com.unity.burst@1.8/manual/csharp-string-support.html
